tests/search/e2e/support/commands/block-editor.js

```diff
@@ -5,18 +5,18 @@ Cypress.Commands.add('openBlockSettingsSidebar', () => {
 			cy.get('.edit-widgets-sidebar__panel-tab').contains('Block').click();
 		} else {
 			cy.get('.edit-post-header__settings button[aria-label="Settings"]').click();
-			cy.get('.edit-post-sidebar__panel-tab').contains('Block').click();
+			cy.get('.edit-post-sidebar__panel-tabs').contains('Block').click();
 		}
 	});
 });
 
 Cypress.Commands.add('openBlockInserter', () => {
-	cy.get('body').then(($el) => {
-		if ($el.hasClass('widgets-php')) {
-			cy.get('.edit-widgets-header-toolbar__inserter-toggle').click();
-		} else {
-			cy.get('.edit-post-header-toolbar__inserter-toggle').click();
+	cy.get('body').then(($body) => {
+		// If already open, skip.
+		if ($body.find('.edit-widgets-layout__inserter-panel-content').length > 0) {
+			return;
```

Comment:
These lines were taken from https://github.com/10up/ElasticPress/blob/develop/tests/cypress/support/commands/block-editor.js#L17-L20 - just in case :-)